### ISSUE TITLE
Wire up miscellaneous data displays

### ISSUE THREAD

**Issue by shanberg** [DecipherNow/gm-fabric-dashboard#215]
- [ ] Avg. Response Time on Summary page
- [ ] Error Rate  on Summary page
- [ ] Cache Size on Settings page
- [ ] Dashboards count on Settings page

**Comment by shanberg** [comment]
@spmcbride1201 I added Uptime here because I realized that jvm/uptime is current returning some fixed value that doesn't appear to be actual uptime milliseconds.

Replacing the readout value with this: `Date.now() - getLatestAttribute(metrics, "jvm/start_time")` gave me a live uptime in milliseconds, and I think using `pretty-ms` instead of `ms` could give us a better visual result.

**Comment by spmcbride1201** [comment]
Okay, I'm actively working this, so I'll take a look.

**Comment by spmcbride1201** [comment]
I think you might be psyching yourself out with the mock. It looks like uptime is the time in milliseconds since the app started. 
![image](https://user-images.githubusercontent.com/8700322/29173480-71c5c374-7db1-11e7-9289-ee6ef43cadcf.png)

**Comment by shanberg** [comment]
Yeah, I think you're right. For the record, this is what I was seeing.
[uptimegraph.mov.zip](https://github.com/DecipherNow/gm-fabric-dashboard/files/1215124/uptimegraph.mov.zip)

I do think we could stand to show the uptime in greater fidelity, at least down to seconds.

**Comment by spmcbride1201** [comment]
I feel like it can get really long and verbose if we just keep it a string and try to make it human readable. Lets roll it into a different issue since there's some different UX things to thing about with it (like do we need a new UI component for showing elapsed time).

**Comment by shanberg** [comment]
Sounds sensible. I'll add a separate issue.
